app/model/exchange/passmarksettings/PassMarkThresholds.scala

```diff
@@ -45,6 +49,15 @@ case class Phase2PassMarkThresholds(
 ) extends PassMarkThresholds with Phase2Thresholds
 
 object Phase2PassMarkThresholds {
-  implicit val phase1PassMarkThresholds = Json.format[Phase2PassMarkThresholds]
-  implicit val phase1PassMarkThresholdsHandler = Macros.handler[Phase2PassMarkThresholds]
+  implicit val phase2PassMarkThresholds = Json.format[Phase2PassMarkThresholds]
+  implicit val phase2PassMarkThresholdsHandler = Macros.handler[Phase2PassMarkThresholds]
+}
+
+case class Phase3PassMarkThresholds(
```

Comment:
It feels redundant to wrap videoInterview in another case class when there is only one field but I guess you want to keep consistency with the other Phase2 and Phase1...